### PR TITLE
Adding command to return subscription usage data 

### PR DESCRIPTION
**Pull request by Matt-Westphal**
Followed the documentation and added working code for retrieving usage. Looks like:
Azure usage list 5/1/2015 5/2/2015

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Matt-Westphal__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: lib/commands/arm/usage/usage._js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+var profile = require('../../../util/profile');
+var utils = require('../../../util/utils');
+
+var usageUtils = require('./usageUtils');
+
+var $ = utils.getLocaleString;
+
+exports.init = function (cli) {
+	var log = cli.output;
```
Comment (yugangw-msft) [inline lib/commands/arm/usage/usage._js]: please fix the indent. If you use the VS Node.js tooling. In the tools->option dialog, set the node.js tab to be 2 spaces

File: lib/commands/arm/usage/usageUtils._js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+
+var utils = require('../../../util/utils');
+
+exports.createUsageManagementClient = function (subscription) {
```
Comment (yugangw-msft) [inline lib/commands/arm/usage/usageUtils._js]: You can remove this file, and let command use the "util.createUsageManagementClient".
If you do have other reasons to keep this file, please rename the file ext to ".js" as you have no async logic here, also the guideline is to minimize the usage of streamline as much as possible.

**Comment by yugangw-msft** [comment]
Please fix CI, and also add a test

**Comment by Matt-Westphal** [comment]
the CI is broken until the node.js package is updated for usage

**Comment by yugangw-msft** [comment]
please submit a PR at sdk-for-node side and publish out a new package. Then update the pcakage.json. I can't merge it with CI failing. The rest looks good to me.

**Comment by Matt-Westphal** [comment]
The PR for sdk-for-node was sent last week, still waiting for it to be merged

**Comment by yugangw-msft** [comment]
Okay, I just now meged

**Comment by Matt-Westphal** [comment]
Thanks!

**Pull request merged**